mmedit/models/common/upsample.py

```diff
@@ -5,6 +5,20 @@
 from .sr_backbone_utils import default_init_weights
 
 
+def pixel_shuffle(x, scale):
+    """Up-sample by pixel shuffle.
+
+    Args:
+        x (Tensor): Input tensor.
+        scale (int): Scale factor.
+
+    Returns:
+        y (Tensor): Output tensor.
+    """
+
+    return F.pixel_shuffle(x, scale)
```

Comment:
We do not need this. We can simply use `self.pixel_shuffle = nn.PixelShuffle(scale)` during initialization.